x/distribution/simulation/operations.go

```diff
@@ -230,7 +229,7 @@ func SimulateMsgFundCommunityPool(txConfig client.TxConfig, ak types.AccountKeep
 		if !hasNeg {
 			fees, err = simtypes.RandomFees(r, ctx, coins)
 			if err != nil {
-				return simtypes.NoOpMsg(types.ModuleName, types.TypeMsgFundCommunityPool, "unable to generate fees"), nil, err
+				return simtypes.NoOpMsg(types.ModuleName, sdk.MsgTypeURL(&types.MsgFundCommunityPool{}), "unable to generate fees"), nil, err
```

Comment:
There should not be any behavior change, actually. We update our tests, and clean-up the code.
I don't think it is breaking.

Reply:
hmm, its api breaking in the sense that it requires users to upgrade code instead of using a newer version and releasing, or does a user not need to make these changes?. I dont see much benefit to backporting this. Are you fine with closing if the above lines are required